src/frontend/src/app/audit-log/audit-log.component.ts

```diff
@@ -11,11 +11,15 @@ export class AuditLogComponent implements OnInit {
   public logSeverityList = [];
   public filterdData = [];
 
-  public limit: Number = 0;
-  public offset: Number = 0;
+  limit = 20;
+  offset = 0;
   public source: String = '';
   public severity: String = '';
 
+  page = 1;             // Default page is 1 in pagination
+  recordCount = 100;    // Total no. of records during pagination (For now it is hard coded, TODO: FOGL-714)
```

Comment:
for asset records too? can not we just have this for audit log?

Reply:
But it is in audit log, not in asset records. (File name is- audit-log.component.ts)